libs/api/domains/municipalities-financial-aid/src/lib/dto/createApplication.input.ts

```diff
@@ -0,0 +1,123 @@
+import { Allow } from 'class-validator'
+
+import { Field, InputType } from '@nestjs/graphql'
+
+import {
+  HomeCircumstances,
+  Employment,
+  ApplicationState,
+  FamilyStatus,
+} from '@island.is/financial-aid/shared/lib'
+
+import { CreateApplicationFileInput } from './createFile.input'
+
+@InputType()
+export class CreateApplicationInput {
+  @Allow()
+  @Field()
+  readonly name!: string
+
+  @Allow()
+  @Field()
+  readonly phoneNumber!: string
+
+  @Allow()
+  @Field()
+  readonly email!: string
+
+  @Allow()
+  @Field(() => String)
+  readonly homeCircumstances!: HomeCircumstances
+
+  @Allow()
+  @Field({ nullable: true })
+  readonly homeCircumstancesCustom!: string
```

Comment:
I have noticed that in our financial-aid api we use ? when nullable: true. Just wondering if that is needed here and other places in this file?

Reply:
The generator doesn't allow the question mark so I have to have nullable in the field 🙃